db/seeds.rb

```diff
@@ -5,3 +5,25 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Post.destroy_all
+Comment.destroy_all
+
+
+
+50.times do |index|
+    created_at = Faker::Date.backward(days: 365 * 5)
+
+    q = Post.create(
+        title: Faker::Company.bs,
+        body: Faker::Lorem.sentence(word_count: 50, supplemental: true),
```

Comment:
Create post fails and throws an error `ActiveRecord::RecordInvalid: Validation failed: Body is too long (maximum is 200 characters)`. You can try this perhaps to avoid running into max length error `body: Faker::Lorem.characters(number: 100)`